packages/tools/kadena-cli/src/devnet/commands/devnetList.ts

```diff
@@ -1,8 +1,9 @@
 import debug from 'debug';
+import type { CreateCommandReturnType } from '../../utils/createCommand.js';
 import { createCommand } from '../../utils/createCommand.js';
 import { displayDevnetsConfig } from '../utils/devnetDisplay.js';
 
-export const listDevnetsCommand = createCommand(
+export const listDevnetsCommand: CreateCommandReturnType = createCommand(
```

Comment:
I just ran `./bin/kadena-cli.js devnet list`, but this doesn't give me the correct volume. 

I start my devnet with this command

```sh
docker volume create kadena_devnet; docker run --rm -it -p 8080:8080 -p 5432:5432 -p 9999:9999 -v kadena_devnet:/data -v ./cwd-extra-migrations:/cwd-extra-migrations:ro --name devnet kadena/devnet
```

and it's using the `kadena_devnet` volume. You can check that by running

```
docker inspect -f '{{ .Mounts }}' devnet
[{volume kadena_devnet /var/lib/docker/volumes/kadena_devnet/_data /data local z true } {bind  /host_mnt/Users/albert/projects/kadena-community/kadena.js-master/packages/apps/graph/cwd-extra-migrations /cwd-extra-migrations  ro false rprivate}]
```

However, `devnet list` doesn't show it:

```
$ ./bin/kadena-cli.js devnet list

executing: bin/kadena-cli.js devnet list 
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
  Name: devnet                                                            
  Port: 8080                                                              
  Volume: N/A                                                             
  Pact folder mount: N/A                                                  
  kadena/devnet version: latest                                           
--------------------------------------------------------------------------------
```